src/modules/45-dashboards/components/DashboardCard/DashboardCard.tsx

```diff
@@ -42,6 +40,13 @@ const DashboardCard: React.FC<DashboardCardProps> = ({
   const { getString } = useStrings()
   const { accountId, folderId } = useParams<{ accountId: string; folderId: string }>()
   const [menuOpen, setMenuOpen] = useState(false)
+  const { editableFolders } = useDashboardsContext()
+
+  const isCloneable = !!editableFolders?.length
```

Comment:
Looks like `editableFolders` isn't optional, it'll aways be an array
```suggestion
  const isCloneable = !!editableFolders.length
```